test/specs/oboe.component.spec.js

```diff
@@ -20,7 +20,7 @@ import { oboe } from '../../src/publicApi';
     if (!Platform.isNode) {
       globalContext = window
     } else {
-      globalContext = GLOBAL || global // eslint-disable-line no-undef
```

Comment:
I was leaving this in so that we could continue to support lower versions of node